test/test_ops.py

```diff
@@ -345,6 +345,115 @@ def test_out(self, device, dtype, op):
         op(*sample.input, *sample.args, **out_kwargs)
         self.assertEqual(expected, out)
 
+    @ops([op for op in op_db if op.aliases])
+    def test_aliases_consistency_eager(self, device, dtype, op):
```

Comment:
What would you think about extending this:

https://github.com/pytorch/pytorch/blob/306420b66e5ab695de99977f481279a9ce764c6e/test/test_ops.py#L203

with the aliases?

Once small challenge with that approach is that it requires updating the inplace checks, like this one:

https://github.com/pytorch/pytorch/blob/627a33125735a0683083f3e2b56e17fee80741b8/test/test_ops.py#L224

to account for inplace aliases, too. This could be done by adding an `is_inplace_variant(self, variant)` method to the OpInfo class that returns True if the variant (including alias variants) is inplace. Or by just checking for `inplace or alias_inplace` (since currently all ops only have one alias). 

This would reduce code duplication and ensure the tests stay consistent in the future.

Reply:
Thanks for the idea ! Yes, such that we shouldn't wait for a refactoring PR. Let me implement that.